theorems/T000142.md

```diff
@@ -3,14 +3,20 @@ uid: T000142
 if:
   and:
   - P000029: true
-  - P000006: true
+  - P000012: true
 then:
   P000061: true
 refs:
-- mr: 2247908
-  name: Products of cozero complemented spaces
+  - zb: "1067.54015"
+    name: Cozero complemented spaces; when the space of minimal prime ideals of a $C(X)$ is compact (Henriksen & Woods)
 ---
 
-Given any cozero $U \subset X$, $X \setminus \overline U$ contains a maximal pairwise-disjoint collection of cozero sets $\mathcal{V}$. $\mathcal{V}$ is countable by CCC, so $\cup \mathcal{V}$ is a cozero complement for $U$.
+Given a cozero set $U$ in $X$, the open set $X\setminus\overline U$
+contains a maximal collection $\mathscr V$ of pairwise disjoint nonempty cozero sets of $X$.
+By {P29} $\mathscr V$ is countable,
+hence $V=\bigcup\mathscr V$ is a cozero set.
+And since $X$ is {P12}, the maximality of $\mathscr V$
+implies that $V$ is dense in $X\setminus\overline U$.
```

Comment:
```suggestion
hence $V=\bigcup\mathscr V$ is a cozero set disjoint from $\overline U$.
And since $X$ is {P12}, the maximality of $\mathscr V$
implies that $U\cup V$ is dense in $X$.
```

Take or leave this suggestion.